src/Lint.jl

```diff
@@ -70,7 +70,12 @@ If file is in base lint all files in base dir.
 function lintpkgforfile{T<:AbstractString}(path::T, ctx::LintContext=LintContext())
     path = abspath(path)
     if ispath(ctx.path)
-        while path != "/"
+        if is_windows()
+            len = length(matchall(r"\\", path))+1
```

Comment:
I removed "+1" and tests work. I think we can removed it. I think I made a mistake yesterday with +1.